content/en/logs/guide/azure-logging-guide.md

```diff
@@ -339,9 +333,28 @@ See [Getting started with Azure Functions][307] for more information.
 [311]: https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#configure-a-connection-string-for-an-azure-storage-account
 [312]: https://docs.datadoghq.com/getting_started/site/
 [313]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=python-v2%2Cisolated-process%2Cnodejs-v4%2Cextensionv5&pivots=programming-language-csharp
+
 {{% /tab %}}
 {{< /tabs >}}
 
+## Advanced configuration
+Refer to the following topics to configure your installation according to your monitoring needs.
+
+### PCI Compliance
+
+<div class="alert alert-warning">
+PCI DSS compliance for APM and Log Management is only available for Datadog organizations in the <a href="/getting_started/site/">US1 site</a>.
+</div>
+
+To set up PCI-compliant Log Management, you must meet the requirements outlined in [PCI DSS Compliance][6]. Send your logs to the dedicated PCI compliant endpoint:
+- `agent-http-intake-pci.logs.datadoghq.com:443` for Agent traffic
+- `http-intake-pci.logs.datadoghq.com:443` for non-Agent traffic
```

Comment:
Just in case you didn't know, you can add these to `assets/scripts/config/regions.config.js` if they need to change depending on region! Not relevant here, but figured I'd mention!